stylesheet.css

```diff
@@ -0,0 +1,74 @@
+body {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  background: #3c3a39;
+  color: white;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  align-self: flex-start;
+}
+
+.hamburger {
+  justify-self: flex-start;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  background: #1c1a19;
+  background-image: url(./images/Back-Ground\ image\ 2.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-attachment: fixed;
+  background-position: right bottom;
+}
```

Comment:
- [ ] Kindly ensure that the background image used on this section is not fixed, in order not to affect other contents that would be added later on.
- [ ] Also, you will need to adjust the position as to match how it appears on the figma design. The background image for this section on the Figma does not overlap the texts. But your design does. Kindly look into that and fix.

### Your Design
![your-bg-img](https://github.com/ommarz/Personal_Portfolio/assets/26597432/e9b5fec0-aec4-41c9-90b5-3ba5d9800884)

### Figma Design
![figma-bg-img](https://github.com/ommarz/Personal_Portfolio/assets/26597432/b6f96be8-0f98-4224-8440-ae775ea2c22c)